### PR TITLE
Act 67 allow uploading file when create or update event

### PR DESCRIPTION
**Pull request by Sang-Tan**

### CODE REVIEW

File: src/main/java/com/dutact/web/configuration/ConverterConfig.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.dutact.web.configuration;
+
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
```
Comment (TranSiTien) [inline src/main/java/com/dutact/web/configuration/ConverterConfig.java]: why do we need an empty class

File: src/main/java/com/dutact/web/features/event/admin/dtos/EventCreateDto.java [hunk 2]
```diff
@@ -1,33 +1,34 @@
...
+import jakarta.validation.constraints.NotNull;
 import lombok.Data;
+import org.springframework.web.multipart.MultipartFile;
 
 import java.time.LocalDateTime;
 
 @Data
 public class EventCreateDto {
-    @JsonProperty("organizerId")
+    @NotNull
```
Comment (TranSiTien) [inline src/main/java/com/dutact/web/features/event/admin/dtos/EventCreateDto.java:11]: why u change from json property to jarkartar validation annotation

File: src/main/java/com/dutact/web/features/event/admin/dtos/EventUpdateDto.java [hunk 3]
```diff
@@ -1,30 +1,31 @@
...
+import jakarta.annotation.Nullable;
 import lombok.Data;
+import org.springframework.web.multipart.MultipartFile;
 
 import java.time.LocalDateTime;
 
 @Data
 public class EventUpdateDto {
-    @JsonProperty("name")
+    @Nullable
```
Comment (TranSiTien) [inline src/main/java/com/dutact/web/features/event/admin/dtos/EventUpdateDto.java:11]: can u add Optional for data type

File: src/main/java/com/dutact/web/features/event/admin/dtos/EventCreateDto.java (hunk 2 above)
Comment (Sang-Tan) [reply src/main/java/com/dutact/web/features/event/admin/dtos/EventCreateDto.java:11]: @ModelAttribute does not use json

File: src/main/java/com/dutact/web/features/event/admin/dtos/EventUpdateDto.java (hunk 3 above)
Comment (Sang-Tan) [reply src/main/java/com/dutact/web/features/event/admin/dtos/EventUpdateDto.java:11]: why?

Comment (Sang-Tan) [reply src/main/java/com/dutact/web/features/event/admin/dtos/EventUpdateDto.java:11]: And do mapstruct and Spring OpenAPI support optional?

File: src/main/java/com/dutact/web/configuration/ConverterConfig.java (hunk 1 above)
Comment (Sang-Tan) [reply src/main/java/com/dutact/web/configuration/ConverterConfig.java]: Deleted

File: src/main/java/com/dutact/web/features/event/admin/dtos/EventUpdateDto.java (hunk 3 above)
Comment (TranSiTien) [reply src/main/java/com/dutact/web/features/event/admin/dtos/EventUpdateDto.java:11]: Hmmm as search than they no support, nvm

**Review by TranSiTien — APPROVED** [review]
LGTM

**Pull request merged**